tutorials/remove all entity shadows

```diff
@@ -0,0 +1,327 @@
+**Add / Remove all shadows by code**
```

Comment:
The top of the document needs to contain "front-matter". This is special json code which will create the actual website. 

You can look at other examples in the tutorials section for what the front-matter look like. You should be able to copy it over into your file quite easily.